sdk/communication/Azure.Communication.Administration/src/Azure.Communication.Administration.csproj

```diff
@@ -7,7 +7,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <EnableApiCompat>false</EnableApiCompat>
     <!-- These supressions should be removed in a production library -->
-    <NoWarn>$(NoWarn);AZC0001</NoWarn>
+    <NoWarn>$(NoWarn);AZC0001;AZC0012</NoWarn>
```

Comment:
Why do we need to disable AZC0012?

Reply:
Otherwise we can not have class Capability

Reply:
Let's use `CodeGenModel` attribute and rename it to `PhoneNumberCapability"

Reply:
Generated code makes class Capability that is eventually causing this error
If I add CodeGenModel and decorate with attribute that will export it as PhoneNumberCapability.
BTW this class used to have name PhoneNumberCapability on the server side and in SDK we used to use CodeGenModel to rename it to Capability.